docs/js-client-sdk.iclientconfig.eventingestionconfig.md

```diff
@@ -1,18 +1,18 @@
 <!-- Do not edit this file. It is automatically generated by API Documenter. -->
 
-[Home](./index.md) &gt; [@eppo/js-client-sdk](./js-client-sdk.md) &gt; [IClientConfig](./js-client-sdk.iclientconfig.md) &gt; [eventTracking](./js-client-sdk.iclientconfig.eventtracking.md)
+[Home](./index.md) &gt; [@eppo/js-client-sdk](./js-client-sdk.md) &gt; [IClientConfig](./js-client-sdk.iclientconfig.md) &gt; [eventIngestionConfig](./js-client-sdk.iclientconfig.eventingestionconfig.md)
 
-## IClientConfig.eventTracking property
+## IClientConfig.eventIngestionConfig property
 
 Configuration settings for the event dispatcher
 
 **Signature:**
 
 ```typescript
-eventTracking?: {
+eventIngestionConfig?: {
         batchSize?: number;
         deliveryIntervalMs?: number;
-        enabled?: boolean;
+        disabled?: boolean;

```

Comment:
I think `enabled` is the most recent option

Reply:
you're right. And it should still be called `eventTracking`, not `eventIngestionConfig`.
I don't know where eventIngestionConfig is coming from on my system, but these doc changes are showing up in all of my PRs